K8/cron-job-deployment.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           restartPolicy: Never
           containers:
           - name: dynamo-backupper
-            image: synicix/dynamo-backupper:latest
+            image: vathes/dynamo-backupper:latest
```

Comment:
Looks good. We should also remove the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` from this manifest now that we have confirmed that there is a proper fallback mechanism.

Reply:
done